model/AI/Vocabularies/EnergyConsumptionDescriptionUnitType

```diff
@@ -0,0 +1,21 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# EnergyConsumptionDescriptionUnitType
+
+## Summary
+
+Specifies the unit of energy consumption
+
+## Description
+
+List different acceptable units of energy consumption. If the unit in which the energy consumption is recorded is not available here, please select other
+
+## Metadata
+
+- name: SafetyRiskAssessmentType
+
+## Entries
+
+- kW-H: Kilo Watt/ hour.
+- MJ: Mega Joule.
```

Comment:
"MJ: Mega Joule." -> "MJ: Megajoule."

Reply:
good point.   +1 on just using full name to avoid ambiguity.